From 70b7207e92a8b7d9d746ea9c1f75cb1b4ee995d2 Mon Sep 17 00:00:00 2001
From: firmwarecostum <145313986+firmwarecostum@users.noreply.github.com>
Date: Wed, 7 Oct 2026 17:19:23 +0800
Subject: [PATCH] Remove untracked files check from build script

Remove the command to list untracked files from the build process.
---
 build/build.ts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build/build.ts b/build/build.ts
index 1ae3f65a..c7897d73 100644
--- a/build/build.ts
+++ b/build/build.ts
@@ -144,7 +144,6 @@ async function getBuildMetadata(): Promise<string> {
   const [commit, diff, newFiles] = await Promise.all([
     execAsync("git rev-parse HEAD"),
     execAsync("git diff HEAD"),
-    execAsync("git ls-files --others --exclude-standard"),
   ]).then((res) => res.map((r) => r.stdout.trim()));
 
   if (!diff && !newFiles) return date + commit.slice(0, 4);
